docs/data-cookbooks/cnm-workflow.md

```diff
@@ -0,0 +1,166 @@
+# Basic CNM Workflow
+
+This entry documents setup of a basic workflow that utilizes the built-in CNM/Kinesis functionality in Cumulus.
+
+Prior to using this entry you should be familiar with the [Cloud Notification Mechanism](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cloud+Notification+Mechanism).
+
+## Configuration
+
+### AWS CLI
+
+Use of this entry assumes you have the [AWS CLI](https://www.google.com/search?q=aws+cli) installed and configured.   If you do not, take a moment to review the documentation and install it now.
+
+### Kinesis
+
+Use of this entry assumes you already have a [Kinesis](https://aws.amazon.com/kinesis/) data steam created for use as a CNM notification stream.
+
+If you do not have a stream setup, please take a moment to review the documentation and setup a basic single-shard stream.
+
+Basic stream manipulation for testing/learning purposes can be accomplished via the [AWS CLI](https://docs.aws.amazon.com/streams/latest/dev/fundamental-stream.html)
+
+For more information on how this process works and how to develop a process that will add records to a stream, read the [Kinesis documentation](https://aws.amazon.com/documentation/kinesis/) and the [developer guide](https://docs.aws.amazon.com/streams/latest/dev/introduction.html).
```

Comment:
Do we provide any sort of leg-up in terms of testing? Or does PO.DAAC have something we could provide?

Reply:
I'm not sure if I understand the context you're using 'testing' in  - the comment on L19 is mostly referring to exploring how to do basic Kinesis stream operations, and it felt like the tutorial covered that (at least for the CLI) fairly nicely.